### DISCUSSION TITLE
Path to `.kryptor` on Linux

### DISCUSSION THREAD

**Discussion by hakavlad** [samuel-lucas6/Kryptor#73] [General]
```
Windows: %USERPROFILE%/.kryptor

Linux: /home/.kryptor

macOS: /Users/USERNAME/.kryptor
```
https://www.kryptor.co.uk/specification 
https://github.com/samuel-lucas6/Kryptor/blob/master/README.md

A small note: `/home/` is not writable (without root capabilities) and contain users' home directories.

In fact, on Linux the key is located in `/home/USERNAME/.kryptor`. The correct ways to describe the location are:

```
~/.kryptor

/home/$USER/.kryptor

$HOME/.kryptor
```

Echoing these paths:

```
$ echo ~/.kryptor
/home/user/.kryptor
$ echo /home/$USER/.kryptor
/home/user/.kryptor
$ echo $HOME/.kryptor
/home/user/.kryptor
```

**Comment by samuel-lucas6** [comment]
Thanks for pointing this out, I've updated the README and website. I think it may have been `~/.kryptor` at one point, but I changed it in case people didn't know what `~` meant. You also can't use `~` with Command Prompt on Windows.
